### ISSUE TITLE
alternate directory 

### ISSUE THREAD

**Issue by bolavefasfas** [AmirulAndalib/MLTB-ALPINE-DOCKER#11]
```
WORKDIR /usr/src/app

RUN chmod 777 /usr/src/app
```

**can I use this instead of the above ? In Dockerfile ?** 

```
WORKDIR /root/downloads 

RUN chmod 777 /root/downloads 
```

**Comment by AmirulAndalib** [comment]
Yes you can modify it accordingly but also don't forget to change the original code directories aswell

**Comment by AmirulAndalib** [comment]
For dockerfile 

https://github.com/AmirulAndalib/MLTB-ALPINE-DOCKER/blob/master/Dockerfile%20Base/Dockerfile#L14-L17

**Comment by bolavefasfas** [comment]
> For dockerfile
> 
> https://github.com/AmirulAndalib/MLTB-ALPINE-DOCKER/blob/master/Dockerfile%20Base/Dockerfile#L14-L17

Thanks
